src/SFA.DAS.EmployerApprenticeshipsService.Application/Mappings/EmploymentAgreementStatusMappings.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Linq;
+using AutoMapper;
+using SFA.DAS.EAS.Application.Dtos;
+using SFA.DAS.EAS.Application.Dtos.EmployerAgreement;
+using SFA.DAS.EAS.Application.Queries.GetAccountEmployerAgreements;
+using SFA.DAS.EAS.Domain.Data.Entities.Account;
+using SFA.DAS.EAS.Domain.Models.Authorization;
+using SFA.DAS.EAS.Domain.Models.EmployerAgreement;
+
+namespace SFA.DAS.EAS.Application.Mappings
+{
+    public class EmploymentAgreementStatusMappings : Profile
```

Comment:
Can this class be added to the `SFA.DAS.EAS.Application.UnitTests.Mappings.WhenAutoMapping` unit tests?